packages/alice/lib/core/alice_core.dart

```diff
@@ -138,69 +139,68 @@ class AliceCore {
       Navigator.push<void>(
         context,
         MaterialPageRoute(
-          builder: (context) => AliceCallsListScreen(this, _aliceLogger),
+          builder: (_) => AliceCallsListScreen(this, _aliceLogger),
         ),
-      ).then((onValue) => _isInspectorOpened = false);
+      ).then((_) => _isInspectorOpened = false);
     }
   }
 
   /// Get context from navigator key. Used to open inspector route.
   BuildContext? getContext() => navigatorKey?.currentState?.overlay?.context;
 
   String _getNotificationMessage() {
-    final calls = callsSubject.value;
-    final successCalls = calls
+    final List<AliceHttpCall> calls = callsSubject.value;
```

Comment:
I'm just thinking about this change - what's the reason of adding type of variable here? Is it for readability or is there any other reason?

Reply:
This question is also for other changes in this PR where you've added type for the variable.

Reply:
Readability and conciseness. It's easier to understand what type of variable is this way without the help of an IDE when reviewing it on Github, for example.